plugins/servicenow/unit_test/util.py

```diff
@@ -0,0 +1,73 @@
+import json
+import logging
+import os
+from icon_servicenow.connection.connection import Connection
+from icon_servicenow.connection.schema import Input
+
+
+class Util:
+    @staticmethod
+    def default_connector(action, connect_params: object = None):
+        default_connection = Connection()
+        default_connection.logger = logging.getLogger("connection logger")
+        if connect_params:
+            params = connect_params
+        else:
+            params = {
+                Input.URL: "https://rapid7.com",
+                Input.CLIENT_LOGIN: {"username": "user1", "password": "mypassword"},
+            }
+        default_connection.connect(params)
+        action.connection = default_connection
+        action.logger = logging.getLogger("action logger")
+        return action
+
+    @staticmethod
+    def read_file_to_string(filename):
+        with open(filename) as my_file:
+            return my_file.read()
+
+    @staticmethod
+    def read_file_to_bytes(filename):
+        with open(filename, "rb") as my_file:
+            return my_file.read()
+
+    @staticmethod
+    def mocked_requests(*args, **kwargs):
+        class MockResponse:
+            def __init__(self, filename, status_code, headers=None):
+                if headers is None:
+                    headers = {"Content-Type": "application/json"}
+                self.status_code = status_code
+                self.headers = headers
+                self.content = Util.read_file_to_bytes(
+                    os.path.join(os.path.dirname(os.path.realpath(__file__)), f"payloads/{filename}.resp")
```

Comment:
Unfamiliar with what this code is doing here - what's the purpose of reading a file in byte representation?

Reply:
Reading in a file as a stream of bytes allows us to interpret the data in whatever format we need to, as opposed to assuming that the data contains a string. In this case, we read in the file `plugins/servicenow/unit_test/payloads/get_attachment_file.resp` as bytes because it contains base64-encoded data.